src/v2/api/controllers/UserController.ts

```diff
@@ -668,7 +771,7 @@ export class UserController {
   @UseGuards(TelegramGuard)
```

Comment:
The order isn't correct. It must be ApiBasicAuth, ApiOkResponse,  ApiBadRequestResponse, ApiUnauthorizedResponse, ApiParam, UseGuards. And add ApiEndpoint with summary of endpoint

Reply:
Not my endpoint